enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/assets/ImageBuilderCommand.java

```diff
@@ -0,0 +1,178 @@
+                            Path tempFile = Files.createTempFile("init", ".gradle");
+
+                            String init = "allprojects {\n" +
+                                    "    afterEvaluate {\n" +
+                                    "        tasks.matching { it.name == 'dockerBuild' }.configureEach {\n" +
+                                    "            images = [\"" + repository.getUrl() + ":$project.version\"]\n" +
+                                    "        }\n" +
+                                    "    }\n" +
+                                    "}";
+
+                            Files.write(tempFile, init.getBytes(StandardCharsets.UTF_8));
+                            RunConfig runConfig = RunUtils.createRunConfig(
+                                    project,
+                                    "",
+                                    "Build container image",
+                                    Collections.emptySet(),
+                                    "--init-script", tempFile.toAbsolutePath().toString(), "dockerBuild", "dockerPush"
+                            );
+                            ExecutorTask task = RunUtils.executeGradle(runConfig, "");
+                            task.waitFinished();
+                            refresh();
+                            result.complete(null);
+                        }
+                        if (r != null && "org-netbeans-modules-maven".equals(r.getProjectType())) { //NOI18N
+                            NbMavenProject nbMavenProject;
+                            final boolean isGdk;
+                            if (ociProject != null) {
+                                nbMavenProject = ociProject.getLookup().lookup(NbMavenProject.class);
+                                isGdk = true;
+                            } else {
+                                nbMavenProject = project.getLookup().lookup(NbMavenProject.class);
+                                isGdk = false;
+                            }
+                            
+                            // Workaround until GCN-4792 is fixed
+                            FileObject projectDirectory = FileUtil.toFileObject(nbMavenProject.getMavenProject().getBasedir());
+                            FileObject pomFile = projectDirectory.getFileObject("pom.xml");
+                            if (pomFile != null) {
+                                pomFile.refresh();
+                            }
+                            
+                            nbMavenProject.getFreshProject().thenAccept(mvnProject -> {
+                                List<String> goals;
+                                String version = mvnProject.getVersion();
+                                if (isGdk) {
+                                    goals = List.of("compile", "deploy", "-pl", "oci", "-Dpackaging=docker", "-Djib.to.image=" + repository.getUrl()+ ":" + version);
```

Comment:
I think this if-else can be replaced by `RunConfig.setReactorStyle(ALSO_MAKE)` that generates `--projects <list> --also-make` into the commandline.